erpnext/patches/v8_3/update_company_total_sales.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2017, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+import frappe
+from erpnext.accounts.doctype.sales_invoice.sales_invoice import update_company_monthly_sales
+
+def execute():
+	'''Update company monthly sales field based on sales invoices this month'''
+	companies = [d['name'] for d in frappe.get_list("Company")]
```

Comment:
@pratu16x7,

Need to reload the company doctype as `update_company_monthly_sales` update's the `total_monthly_sales` field in company

Also add the patch entry in patches.txt